packages/data-portal-commons/src/components/ExpandableText.tsx

```diff
@@ -25,13 +26,17 @@ export const ExpandableText: React.FunctionComponent<ExpandableTextProps> = (
     props
 ) => {
     // truncate text by default
-    const [truncated, setTruncated] = useState<boolean>(true);
+    const [truncated, setTruncated] = useState<{ [key: string]: boolean }>({});
```

Comment:
It's confusing for this component to keep the state of multiple values when it only receives a single text value.